src/DssSpell.sol

```diff
@@ -42,7 +66,48 @@ contract DssSpellAction is DssAction {
     function actions() public override {
         // ----- Deploy Multiswap Conduit for RWA015-A -----
 
+        // ----- Add previous CRON KEYS -----
```

Comment:
This should be added to the executive sheet to be included into the spell, currently I see no mentioning of CRON

Also (if the spreadsheet is updated) I suggest to move this section beneath the flapper deployment, so there is just one section related to cron and it's not spread throughout the spell

Reply:
The thing is flapper has to add a cron job to operate in the keepers network. Problem is pause proxy authorization to the sequencer happened not very long time ago, that's why there are not keys in the chainlog yet. We can opt to add the cron to the sequencer and forget about chainlog keys but I'd say if the pause proxy will be executing actions in a contract, that one should be added to the chainlog. This is only for mainnet.
GovAlpha should put this in the copy for sure.

Reply:
Fine with me if it's part of the copy with exact addresses that goes to the chainlog + description of adding flapper to the keepers network. Would you please contact govalpha to include those?

Reply:
I can ping govalpha.